cmd/statusd/main.go

```diff
@@ -106,6 +110,12 @@ func main() {
 		}
 	}
 
+	// Check if profiling shall be enabled.
+	if *pprofEnabled {
+		pprof := profiling.NewProfiler(*pprofPort)
+		pprof.Run()
```

Comment:
One thing that came to my mind. Does it stop properly when SIGINT `ctrl+c` is sent? When I was working on the stats server, there was a problem with that and SIGINT did not stop the process because this signal is handled in the process actually.

Reply:
My local tests yesterday showed no problems. Ending `statusd` with `ctrl-c` always worked properly.